manifests/server/rsync_shares.pp

```diff
@@ -57,26 +57,26 @@
     keys($rsync_environments).each |String $_env| {
 
       # Do the Global items first
-      if $rsync_environments[$_env]['rsync']['global'] {
+      if $rsync_environments[$_env]['rsync']['Global'] {
         $_globals_dir = 'rsync/Global'
 
-        if 'clamav' in $rsync_environments[$_env]['rsync']['global']['shares'] {
+        if 'clamav' in $rsync_environments[$_env]['rsync']['Global']['shares'] {
```

Comment:
This probably won't work. On live systems, the keys are forced lowercase but not during spec testing.